grafana-plugin/src/pages/users/Users.tsx

```diff
@@ -75,10 +75,6 @@ class Users extends React.Component<UsersProps, UsersState> {
     const { usersFilters, page } = this.state;
     const { userStore } = store;
 
-    if (!store.isUserActionAllowed(UserAction.ViewOtherUsers)) {
```

Comment:
but if user really doesn't have such permissions then he/she will able to see users list...it seems we need to solve this issue in some other way